man/systemd-stub.xml

```diff
@@ -77,7 +77,16 @@
       <listitem><para>A <literal>.splash</literal> section with an image (in the Windows
       <filename>.BMP</filename> format) to show on screen before invoking the kernel.</para></listitem>
 
-      <listitem><para>A <literal>.dtb</literal> section with a compiled binary DeviceTree.</para></listitem>
+      <listitem><para>Zero or more <literal>.dtb</literal> sections. If only one such section is present, it
+      will always be passed to the kernel. If there at least two of them, stub will first try to match the
+      DeviceTrees bundled into UKI by comparing the first <literal>compatible</literal> string in each bundled
+      DeviceTree with the first <literal>compatible</literal> string supplied by the firmware. If that fails
+      and <literal>.hwids</literal> section exists, it will be used for matching process instead. The stub
+      will use some SMBIOS values to calculate hardware IDs of the machine (that is, it will calculate hashes of
+      different combinations of these values, the same thing what <literal>fwupdtool hwids</literal> will do), then it will
```

Comment:
i wouldn't reference fwupdtool here, but the microsoft spec behind it, i.e. probably a link like this:

https://learn.microsoft.com/en-us/windows-hardware/drivers/install/specifying-hardware-ids-for-a-computer